src/test/java/Problem2Test.java

```diff
@@ -1,14 +1,28 @@
+
 import org.junit.Test;
 
 import static org.junit.Assert.*;
 
-public class Problem2Test {
+public class Test2{
 
     // Example test
     // Okay to change
     @Test
-    public void testBubbleSort() {
+    public boolean testBubbleSort() {
         assertTrue(false);
```

Comment:
this line will always fail your test. remove it so the rest of your tests get a chance to run.

in your test you should use assertArrayEquals. See how test is done in problem 1 for an example.